compte/management/commands/purge_inactive_accounts.py

```diff
@@ -39,7 +39,7 @@ def handle(self, *args, **options):
             get_user_model()
             .objects.annotate(erps_count=Count("erp"))
             .filter(
-                is_active=False,
+                last_login=None,
```

Comment:
Have you considered using a soft-delete package like this one? https://pypi.org/project/django-soft-delete/

Reply:
That raise an interesting point: should we consider the user as deleted, or as an anonymized user. The is_active field was used here as an non-actived account, but to it's not the primary use

Reply:
What is the primary usage of the `is_active` field? 🤔